src/settings/LanguagePicker.js

```diff
@@ -57,7 +50,7 @@ export default class LanguagePicker extends PureComponent<Props> {
 
     return (
       <FlatList
-        ItemSeparatorComponent={() => <View style={styles.separator} />}
+        ItemSeparatorComponent={() => <OptionDivider />}
```

Comment:
This can be simplified to
```jsx
        ItemSeparatorComponent={OptionDivider}
```
in this context; there's no need for a function wrapper here.